recipes/rockpool/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "rockpool" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b3ded42eba5e25abfda55f839ffe983010f423806dbfca961cba2321c00dc6c8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - numpy
+    - scipy
+    - numba
+    - python >=3.6
+
+test:
+  imports:
+    - rockpool
+
+about:
+  home: https://gitlab.com/ai-ctx/rockpool
+  license: AGPL-3.0
```

Please add a license family.
```suggestion
  license: AGPL-3.0
  license_family: AGPL
```